giving-presentations/giving-presentations.md

```diff
@@ -114,4 +114,9 @@ Try crafting a 5-minute technical presentation about a homework assignment or pr
 
 ### Check for Understanding
 
-Practice makes perfect: have students write and deliver speeches, and practice giving and receiving feedback on their performance.
+- How important is it to make good presentations for your career?
+- How to create a good outline?
+- Important points for creating good slides?
```

Comment:
```suggestion
- What are Important things to remember for creating good slides?
```

Reply:
Thanks for the review, I saved the suggestions